package.json

```diff
@@ -65,7 +65,7 @@
     "node": "^14.16.0 || >=16.0.0"
   },
   "lint-staged": {
-    "*.js": "eslint --ignore-path .gitignore --cache --fix",
+    "!(packages/*/tests/**/fixtures/**/*)*.+(j|t)s": "eslint --ignore-path .gitignore --cache --fix",
```

Comment:
Adds support for ts files and ignores fixtures, and does not execute for fixtures. The eslint ignore does not help we have to ignore them here.